src/environment.F

```diff
@@ -800,6 +802,10 @@ SUBROUTINE read_global_section(root_section, para_env, globenv)
          CALL m_cpuinfo(model_name)
          WRITE (UNIT=output_unit, FMT="(T2,A)") &
             start_section_label//"| CPU model name : "//TRIM(model_name)
+         IF (m_cpuid_static() .LT. m_cpuid()) THEN
+            CPWARN("the compiler target flags used to build this binary are insufficiently"//
```

Comment:
Can we make this a warning which does not increase the warning counter? (maybe introduce a new category `hints`?)
Since this will be triggered in heterogeneous environments and most likely any Linux distro (possibly as well as CP2K built with Spack since we're using `-mtune=native` there instead of `-march=...`), this will lead to people start ignoring the warning counter again and hamper automated calculations and output analysis since we have to filter by specific warnings again to separate real warnings which can significantly affect the result and mere hints (which don't).

Reply:
Sure, will do!

Reply:
Finished.